router/router.go

```diff
@@ -2,11 +2,11 @@ package router
 
 import (
 	"errors"
+	"html/template"
```

Comment:
This breaks the application.

I'll open a new pull request building on your work and attributing you with a co-authorship.

Reply:
Here is the PR I verified it works well.  https://github.com/ngoduykhanh/wireguard-ui/pull/673